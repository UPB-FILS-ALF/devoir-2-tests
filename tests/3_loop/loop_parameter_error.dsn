& LOOP 2 parameters

POSITION: 120 100
LOOP: 1 2
CIRCLE: 50 200 100 
RECTANGLE: 20 100 40 100
REPEAT
LINE:  500 500 location