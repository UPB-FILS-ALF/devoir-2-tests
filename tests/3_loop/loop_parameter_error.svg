<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- position 120, 100 -->
	<circle cx="50" cy="200" r="100" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(255,255,255)"/>
	<polygon points="20,100 40,100 40,100 20,100" stroke="rgb(0,0,0)"  stroke-width="1"  fill="rgb(255,255,255)"/>
	<line x1="120" y1="100" x2="500" y2="500" stroke="rgb(0,0,0)"  stroke-width="1" />
</svg>
