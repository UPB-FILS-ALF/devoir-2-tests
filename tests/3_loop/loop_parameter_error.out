LINE 4 HAS ERROR: LOOP has 1 parameters, you wrote 2
LINE 7 HAS ERROR: REPEAT and no LOOP
